webapp/src/toolbox.tsx

```diff
@@ -685,7 +679,7 @@ export interface ToolboxCategory {
 
     customClick?: (theEditor: editor.ToolboxEditor) => boolean;
     advanced?: boolean; /*@internal*/
-    isExtension?: boolean;
+    delete?: boolean;
```

Comment:
`delete` is a reserved word. What about `deletable` or `allowDelete`.

Reply:
Changed to allowDelete